### DISCUSSION TITLE
Is there any easy way to share my Quasar app footer with a static website?

### DISCUSSION THREAD

**Discussion by gillisig** [quasarframework/quasar#10377] [General - Components / Directives / etc]
Hi guys,

Is there any easy way to share a footer from a Quasar app to a static website? I'm running my blog using Eleventy and would optimally like to share the footer of my Quasar app with my blog, which just contains basic links and such. 

Thanks,
Gilli

**Comment by ontwikkelfabriek** [comment]
@gillisig 

> Is there any easy way to share a footer from a Quasar app to a static website? 

Maybe with an iframe or some JQuery magic ( probably not going to work anymore). It's a bad practice to try to do what you want ( trying to solve this problem client side). A better way would be to load the footer **server** **side** and then include it in the output of your own site html.

**Reply by gillisig** [reply]
Yeah I'm not really thinking about it in terms of doing it on the front-end, rather maybe some build process that would automate it.

**Reply by ontwikkelfabriek** [reply]
@gillisig 

I hope you realize when you say `rather maybe some build process...`, it will have to be the build process of the static site, where you want to include the header. (Not the Quasar app that contains the header)
